src/styles/_card.scss

```diff
@@ -0,0 +1,120 @@
+.catalog {
+  display: grid;
+  justify-items: center;
+  grid-template-columns: 1fr;
+  grid-gap: 48px 46px;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+.catalog-card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: color 300ms;
+  &_title {
+    font-size: 12px;
+    margin-bottom: 4px;
+    font-weight: 500;
+    line-height: 1.5;
+    transition: color 300ms;
+  }
+  &:hover {
+    .catalog-card_title {
+      color: #34568b;
+    }
+  }
+  &_image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+  &_subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondaryColor;
+    margin-bottom: 16px;
+  }
+  &_button {
+    width: 166px;
+    height: 40px;
```

Comment:
The fixed width and height for the .catalog-card_button might not be responsive. Consider using relative units or media queries to adjust these properties for different screen sizes.